docs/operations/activating_feature_gates.md

```diff
@@ -0,0 +1,34 @@
+# Activating feature gates
+
+KubeVirt has a set of features that are not mature enough to be enabled by
+default. As such, they are protected by a Kubernetes concept called
+[feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
+
+## How to activate a feature gate
+You can activate a specific feature gate directly in KubeVirt's CR, by
+provisioning the following yaml, which uses the "LiveMigration" feature gate
+as an example:
+```bash
+cat << END > enable-feature-gate.yaml
+---
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    developerConfiguration: 
+      featureGates:
+        - LiveMigration
+END
+
+kubectl apply -f enable-feature-gate.yaml
+```
+
+**Note:** the name of the feature gates is case sensitive.
+
+## List of feature gates
+The list of feature gates (which evolve in time) can be checked directly from
+the [source code](https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-config/feature-gates.go#L26).
```

Comment:
Is there a way to show a "preview" of the link in markdown ? IMO that would be helpful - I'd rather not have to maintain the list of gated features in this file.

Reply:
Absolutely.  See README in the main repo

The basic process is ...
1. run `make run`
2. edit markdown
3. web browse to http://0.0.0.0:8000
4. review edit
5. git commit and push
6. create pull req

Reply:
re FeatureGate list .... @davidvossel is there a better way we can maintain this list?

Reply:
I completely misunderstood.  Sorry about that.

Try out the info from this post.... https://stackoverflow.com/questions/41966638/jekyll-on-github-pages-include-markdown-in-another-markdown-file

Reply:
My bad; shouldn't have referred the word "preview". That was awfully misleading  :)

> Try out the info from this post.... https://stackoverflow.com/questions/41966638/jekyll-on-github-pages-include-markdown-in-another-markdown-file

I've looked at the link you've pasted, and afaiu, this would only apply when including a file relative to the one doing the including. Which, I guess means it must be included in the same repo.

It seems that we either link to the source code, or just manually keep the list in sync ... 

Is there any established precedence here ? What are we doing elsewhere in the docs ? I am OK with either option, but would prefer to not have to keep the list synced manually.

Reply:
I like how you have found the other references to feature gate and linked to this page.  Let's call the link to the code good enough and move on.  Maybe later we can think of a way to import that list.